packages/api/src/middlewares/withDate.ts

```diff
@@ -21,10 +20,12 @@ export const withDatePeriod = t.middleware(({ next, input }) => {
   const { from, to } = datePeriodSchema.parse(input) || {};
   const hasAtLeastOneDate = from || to;
 
-  const datePeriod: DatePeriod | undefined = hasAtLeastOneDate && {
-    from: from ? normalizeDate(from) : undefined,
-    to: normalizeDate(to ?? dayjs()),
-  };
+  const datePeriod =
```

Comment:
If hasAtLeastOneDate is false, then datePeriod will be a boolean. Is it expected?

Reply:
If `datePeriod` is a boolean is gonna be `false` and then is gonna be treated as it's not defined, but to make it more clearer let's set it to `undefined` instead.